### ISSUE TITLE
Multiple variable setters in .env file

### ISSUE THREAD

**Issue by deitry** [kondratyev-nv/vscode-python-test-adapter#88]
Hello!
I have a $PYTHONPATH in .env which contains multiple paths.
If I write it like this:
```
PYTHONPATH="<path1>:<path2>:<path3>"
```
everything works as expected. But it looks messy so I tried to rewrite it like this:
```
PYTHONPATH="${PYTHONPATH}:<path1>"
PYTHONPATH="${PYTHONPATH}:<path2>"
PYTHONPATH="${PYTHONPATH}:<path3>"
```
Python autocomplete feature finds this paths and handle them correctly, but it seems that PythonTestAdapter cannot parse .env, it shows no tests at all. If I put them back in single line, tests are shown again.

**Comment by kondratyev-nv** [comment]
Thank you for the issue! I'll take a look.

**Comment by kondratyev-nv** [comment]
@deitry I've just published new version 0.3.8 with the fix for variable resolution in .env file. Please, update and let me know if the issue persists.
